test/functionalTest/cases/1587_q_query_has_type/q_query_has_type.test

```diff
@@ -0,0 +1,192 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Although I undesrtand that having the test is working as it is now due to some steps are commented, I would suggest to rename it to .pending1675.DISABLED in order to make explicit when this test could be "recovered".